<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="cmb_name_Notes"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item>
             <widget class="QTextEdit" name="te_notes_Main"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_clockin_Notes">
            <property name="text">
             <string>Clock In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_clockout_Notes">
            <property name="text">
             <string>Clock Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_submit_Notes">
          <property name="text">
           <string>Submit Note</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Company Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmb_name_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Phone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_phone_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_email_Edit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_addy_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>City</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_city_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>State</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_state_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Zip</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_zip_Edit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>MainContact</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_contact_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>WebSite</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_website_Edit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Default Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateD_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Software Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateS_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>IT Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateI_Edit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Database Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateDB_Edit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_Update_Client">
            <property name="text">
             <string>Update Client</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>User Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_user_Login">
                <property name="maximumSize">
                 <size>
                  <width>329</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_pw_Login"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Host</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_host_Login"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_connect_Login">
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_20">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Company Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_name_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_21">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Phone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_phone_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_22">
              <item>
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_email_Edit_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_23">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_addy_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_24">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>City</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_city_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_25">
              <item>
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>State</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_state_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_26">
              <item>
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Zip</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_zip_Edit_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_27">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>MainContact</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_contact_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_28">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>WebSite</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_website_Edit_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_29">
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Default Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateD_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_30">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Software Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateS_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_31">
              <item>
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>IT Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateI_Edit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_32">
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>Database Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_rateDB_Edit_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_Update_Client_2">
            <property name="text">
             <string>Add Client</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>191</width>
          <height>58</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_33">
         <item>
          <widget class="QComboBox" name="cmb_name_Delete"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete_Delete">
           <property name="text">
            <string>Delete Client</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDelta_Client">
    <property name="title">
     <string>Delta Client</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd_Client"/>
    <addaction name="actionRemove_Client"/>
    <addaction name="actionEdit_Client"/>
    <addaction name="actionNotes"/>
   </widget>
   <addaction name="menuDelta_Client"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Client">
   <property name="text">
    <string>Add Client</string>
   </property>
  </action>
  <action name="actionRemove_Client">
   <property name="text">
    <string>Remove Client</string>
   </property>
  </action>
  <action name="actionEdit_Client">
   <property name="text">
    <string>Edit Client</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
